packages/utilities/psammead-assets/src/svgs/mediaIcons.jsx

```diff
@@ -61,17 +70,29 @@ const mediaIcons = {
       viewBox="0 0 32 26"
       width="16px"
       height="13px"
-      focusable="false"
+      {...defaultAttrs}
     >
       <path d="M9,2V0H4V2H0V26H32V2ZM6.5,10A2.5,2.5,0,1,1,9,7.52,2.5,2.5,0,0,1,6.5,10ZM20,23a9,9,0,1,1,9-9A9,9,0,0,1,20,23Z" />
       <circle cx="20" cy="14.02" r="5.5" />
     </PhotoMediaIcon>
   ),
   guidance: (
-    <GuidanceIcon viewBox="0 0 32 32" width="32" height="32" focusable="false">
+    <GuidanceIcon viewBox="0 0 32 32" width="32" height="32" {...defaultAttrs}>
       <path d="M32,16A16,16,0,1,1,16,0,16,16,0,0,1,32,16Zm-8.6-.8H16.6v3.1h2.9c-.2,1.5-1.6,2.4-3.7,2.4s-3.7-2.4-3.7-4.7,1-4.7,3.7-4.7,3,.8,3.2,2.2h4.2c-.4-4-3.9-5.8-7.2-5.8-4.8,0-8,3.7-8,8.3s3.1,8.3,8,8.3a5.4,5.4,0,0,0,4.5-2.1l.2,1.7h2.7Z" />
     </GuidanceIcon>
   ),
+  clock: (
+    <ClockIcon
+      viewBox="0 0 13 13"
+      width="13"
+      height="13"
+      aria-hidden="true"
```

Comment:
I completely understand why this attribute has been added, because we're constantly wrapping our icons in containers, which have this attribute. 😅 However:
- This affords us more control within Simorgh, when it comes to choosing what's hidden from assistive technology and what's not.
- If we're adding this to one icon, we should probably add it to the rest for consistency and that's probably a discussion we should have with the wider team.

Reply:
From my conversation with @greenc05 I understand that all icons should have `aria-hidden="true"` attribute, so I will add them to other media icons too.

Reply:
As long as asssistive tech can't access the SVG I don't think it matters if aria-hidden true is applied to a container wrapping the SVG or directly to the SVG tag, personally I would prefer to apply it directly to the SVG as that's what I've tested and know it works, though if it's better from a code writing point of view applying it to a wrapping container is also fine - no one as flagged any issues from swarms doing it this way

Reply:
My worry is that adding aria-hidden at this level, rather on a case by case basis will tie our feet together. What if we have an accessible icon that uses a [title](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) element? We use the container > icon pattern everywhere we use an icon in our code at the moment.

Reply:
We don't use the title element anywhere, using visually hidden text has greater support

Reply:
If our icons will always be hidden from assistive tech. then this could be a good route to go down.

Reply:
I can't guarantee that we will always hide our icons but that's what I recommend currently...